### PR TITLE
release, and readme update to make the warning a little less scary

### PR DESCRIPTION
**Pull request by boutell**

### CODE REVIEW

**Comment by boutell** [comment]
This became a more ambitious update of the README, so I've assigned to both Val and Bea for review. Let me know if I missed anything.

File: README.md [hunk 1]
```diff
@@ -1,48 +1,64 @@
...
 ```
 
-Notice that the `Unsplash` Api require to create a free developer account which will provide you an access key.
+Note: to access the `Unsplash` API you will need to create a free developer account, which will provide you with an access key. For Wedia you will need to work with the Wedia team.
+
+If you prefer not to store keys in your source code, you can use an environment variable, like this:
 
-Alternatively you may use an environment variable, to avoid storing keys in your source code.
+```javascript
+        accessKey: process.env.UNSPLASH_ACCESS_KEY
```
Comment (abea) [inline README.md:44]: hella indented

File: README.md [hunk 2]
```diff
@@ -65,16 +81,18 @@ Each module must have a `mediaSourceConnector` option :
...
+
+For custom filters, you must provide `label` and `type`, which currently may only be `select`.
+
+A connector must have at least three methods declared in its `construct`. These will be called by `apostrophe-media-sources`:
 
-* `self.find` (req, filters):
-  You get gere the filters selected by the user.
-  This one must get the data from the provider and format it, it has to return
-  Images, this is an array of images which have been formatted this way:
+* `async find (req, filters)`:
```
Comment (abea) [inline README.md:94]: Are these _not_ placed on the `self` object?

File: README.md [hunk 3]
```diff
@@ -93,35 +111,68 @@ those will be called by `apostrophe-media-sources`:
...
-```js
-'apostrophe-media-sources-wedia': {
-  script: {
-    src: 'https://xxx.wedia-group.com/asset-picker/wedia-content-picker.js',
-    server: 'https://xxx.wedia-group.com',
-    name: 'WediaContentPicker',
-    maxConcurrentImports: 20 // optional - activates the multiple import - if absent, only single import will be available
+Configuration of your module in `index.js` will then look like:
+
+```
```
Comment (abea) [inline README.md:168]: Did you mean to remove the syntax indicator from ```js?

**Review by abea — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,48 +1,64 @@
...
-This module will override `apostrophe-images`, adding the possibility to browse different images providers.
+You need to already have an Apostrophe project in which you will install `apostrophe-media-sources`. This module will override `apostrophe-images`, adding the possibility to browse different image providers.
+
+For now, `apostrophe-media-sources-unsplash``apostrophe-media-sources-wedia` have been implemented. These modules are part of the `apostrophe-media-sources` bundle, so you don't need to separately install them.
 
-For now, only `apostrophe-media-sources-unsplash` has been implemented, this module is part of the `apostrophe-media-sources` bundle, so you don't need to install it.
+## Warnings
 
-This module will be available on `npm` soon, for now, you can still clone it to have a look.
+**When using this module, your site cannot be edited in the IE11 browser.** Modern Microsoft Edge browsers work just fine. This site does not affect the experience of ordinary logged-out site visitors in IE11.
```
[the remark this answers is not in the document]
Comment (boutell) [reply README.md]: Clarified

File: README.md [hunk 5]
```diff
@@ -1,48 +1,64 @@
...
     modules: {
-    'apostrophe-media-sources': {},
-    'apostrophe-media-sources-unsplash': {
-      accessKey: 'my-access-key'
-    },
+      'apostrophe-media-sources': {},
+      'apostrophe-media-sources-unsplash': {
+        accessKey: 'my-access-key'
+      },
+      'apostrophe-media-sources-wedia': {
```
[the remark this answers is not in the document]
Comment (boutell) [reply README.md:25]: It's not an OR, you may use one or the other or both or write more etc. So I don't think there's any appropriate language there, it's just a couple o' modules hangin' out.

File: README.md (hunk 2 above)
Comment (boutell) [reply README.md]: The `async` keyword made me crazy? I think I've fixed this, by showing that expectation while still using the actual valid syntax.

**Review by abea — APPROVED** [review]

**Review by ValJed — APPROVED** [review]
Thanks for the improvements!

**Pull request merged**